addons/equipment/cfgUnitInsignia.hpp

```diff
@@ -0,0 +1,53 @@
+class CfgUnitInsignia
+{
+	class ga_insignia_de
+	{
+		displayName = "Germany"; // Name displayed in Arsenal
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "\x\grad\addons\equipment\data\insignia\de_v2.paa"; // Image path
+		textureVehicle = ""; // Does nothing currently, reserved for future use
+	};
+	
+	class ga_insignia_es
+	{
+		displayName = "Spain"; // Name displayed in Arsenal
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "\x\grad\addons\equipment\data\insignia\es_v2.paa"; // Image path
+		textureVehicle = ""; // Does nothing currently, reserved for future use
+	};
+	
+	class ga_insignia_it
+	{
+		displayName = "Italy"; // Name displayed in Arsenal
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "\x\grad\addons\equipment\data\it_v2.paa"; // Image path
```

Comment:
insignia folder is missing in path

Reply:
This is the actual error in the path. Here and on all insignia below this point.

Reply:
🤦‍♂️ My Copy & Paste skills = -9001 
fixed:
https://github.com/gruppe-adler/gruppe_adler_mod/pull/90/commits/73f2b05e47be7884d19706579b8de19ac4e4a628